application/src/main/java/org/togetherjava/tjbot/commands/help/ChangeHelpCategoryCommand.java

```diff
@@ -71,10 +71,6 @@ public ChangeHelpCategoryCommand(@NotNull Config config, @NotNull HelpSystemHelp
     public void onSlashCommand(@NotNull SlashCommandInteractionEvent event) {
         String category = event.getOption(CATEGORY_OPTION).getAsString();
 
-        if (!helper.handleIsHelpThread(event)) {
-            return;
-        }
-
```

Comment:
Now it could also be used in the overview channel itself, not an option. I.e. we have to at least check that its a thread and not a regular channel.

Reply:
normal members can't send messages in overview channel